new Tags chapter and Custom Fields chapter

FlowWriter opened the pull request:
This includes new Tags chapter and Custom Fields chapter. Also a little bit of Tenants chapter. 

I also had to fix all the image references to include the relative portion of the path name. Hope I did it right!

Commit:
fixing includes syntax for Kaui Guide

Commit:
writing Tags section

Commit:
added basic info for Custom Fields

Commit:
writing Tenants section

Commit:
write Tags chapter

Commit:
Merge branch 'killbill:v3' into kaui-only-9

Commit:
writing Tags chapter

Commit:
writing Tags chapter

Commit:
housekeeping

Commit:
housekeeping

Commit:
tweaking Tags overview

Commit:
writing Custom Fields

also tweaking Tags

Commit:
write Tenants chapter

Commit:
removed tag info from Accounts chapter

this info is covered in the new Tags chapter

Commit:
writing Custom Fields chapter

Commit:
move Tenants chapter in front of Users chapter

also removed "in process" from unfinished sections

Commit:
tied up loose ends in Tags chapter

Commit:
writing Custom Fields chapter

Commit:
Merge branch 'killbill:v3' into kaui-only-9

Commit:
writing Custom Fields

Commit:
writing Custom Fields

Commit:
fixing image references

sbrossie commented:
@FlowWriter Could you attach the PDF - if you have it?

reshmabidikar commented:
Hi @FlowWriter, I see that there is a [tags.pdf](https://github.com/FlowWriter/killbill-docs/blob/kaui-only-9/userguide/kaui/includes/tags.pdf) but it appears to be different from [tags.adoc](https://github.com/FlowWriter/killbill-docs/blob/kaui-only-9/userguide/kaui/includes/tags.adoc). So, perhaps `tags.pdf` is an older version and needs to be updated? Secondly, the pdf file for [custom-fields.adoc](https://github.com/FlowWriter/killbill-docs/blob/kaui-only-9/userguide/kaui/includes/custom-fields.adoc) is missing, could you please add that too?

ghost commented:
@sbrossie and @reshmabidikar - Here is the PDF of the entire document. You only need to check the Tags and Custom Fields chapters.
[Kaui_Guide_Draft.pdf](https://github.com/killbill/killbill-docs/files/8254025/Kaui_Guide_Draft.pdf)

sbrossie commented:
@FlowWriter Looking great, I just have one comment/question: On Page 76, there is a screenshot for the Account#tags, but you are describing tags associated with an invoice - should it be removed?

reshmabidikar commented on userguide/kaui/includes/tags.adoc:
```diff
@@ -1 +1,127 @@
+Kill Bill has two types of tags:
+
+* *Control tags*&#8212;Also referred to as _system tags_, these tags modify the behavior of the system. In Kaui, you can view and attach these tags, but you cannot add, change, or delete the underlying <<Add Tag Definition, tag definition>>. Kill Bill includes several control tags; see the API documentation for https://killbill.github.io/slate/#tag[a list].
+
```
Looks like the [Account Tags](https://killbill.github.io/slate/#account-tags) section lists all the tags. I will move this to the [Tag](https://killbill.github.io/slate/#tag) section in the slate docs.

reshmabidikar commented on userguide/kaui/includes/tags.adoc:
```diff
@@ -1 +1,127 @@
+
+[NOTE]
+*Note:* You can use the API to attach tags to even more object types, such as payments and subscriptions. For more information, see the "Tags" section of the relevant object type section (for example, Payments) in the https://killbill.github.io/slate/[API Reference].
+
```
Kaui also allows adding a user tag to a subscription. See attached screenshot.

<img width="876" alt="Subscription_Tag" src="https://user-images.githubusercontent.com/85998496/158516914-07154745-9f42-4ac8-add4-aea216cd0931.png">

FlowWriter replied:
See page 76 of PDF.

reshmabidikar commented on userguide/kaui/includes/tags.adoc:
```diff
@@ -1 +1,127 @@
+. Click *Tags* on the sub-menu.
+
+Kaui opens the Tags page and displays only the tags assigned to that account or its related invoice(s).
+
```
Kaui also displays tags associated with subscriptions (if any).

FlowWriter replied:
I edited the section to indicate the process was the same across accounts, invoices, and subscriptions.

reshmabidikar commented on userguide/kaui/includes/custom-fields.adoc:
```diff
@@ -0,0 +1,106 @@
+Kaui displays the Custom Fields page:
++
+image::../assets/img/kaui/CustomFields-Populated.png[width=850,align="center"]
++
+. On the custom field row, click *Destroy*.
```
Is this correct? I do not see a **Destroy** link in the custom field row.

FlowWriter replied:
@reshmabidikar - Stéphane said that functionality appears to be missing from Kaui. You don't see "Destroy" when you're logged in, do you?

FlowWriter replied:
I have removed the "deleting" section from the Custom Fields chapter.

reshmabidikar reviewed (commented):
Looks good, I've added a few minor comments

ghost commented:
> Looking great, I just have one comment/question: On Page 76, there is a screenshot for the Account#tags, but you are describing tags associated with an invoice - should it be removed?

@sbrossie - In the beginning, I explain that the steps are for applying a tag to accounts _and_ invoices. I can edit the steps to be more clear.

Commit:
corrections per review process

you'll see a lot of image ref changes because I have to do it a certain way for the PDF compile, then another way for our adoc to HTML compile

Commit:
Merge branch 'killbill:v3' into kaui-only-9

Commit:
deleted section on deleting custom fields

Commit:
regenerate PDF

Commit:
changing all image refs back to what they should be

fingers crossed

ghost commented:
Uploading latest PDF
[Kaui_Guide_Draft.pdf](https://github.com/killbill/killbill-docs/files/8307387/Kaui_Guide_Draft.pdf)

ghost commented:
I have uploaded the latest PDF, so I'm hoping this is ready to merge. :-)

sbrossie commented:
Merging...  @reshmabidikar If somehow if you feel like your comments were not correctly addressed, chime in, we can always open a new PR.
